src/styles/style.css

```diff
@@ -0,0 +1,48 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.clock {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+}
+
+.clock__box {
+  border-radius: 50%;
+  width: 400px;
+  height: 400px;
+  border: 4px black dotted;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.clock__box-center {
+  width: 30px;
```

Comment:
there are some duplicated size, like 30px, 400px, etc.
so, make sense to move all these duplicated sizes to the variables